src/OpenTelemetry.Api/Metrics/BoundCounterMetric.cs

```diff
@@ -28,9 +28,9 @@ public abstract class BoundCounterMetric<T>
         /// <summary>
         /// Adds the given value to the bound counter metric.
         /// </summary>
-        /// <param name="context">the associated <see cref="SpanContext"/>.</param>
+        /// <param name="context">the associated <see cref="SpanReference"/>.</param>
         /// <param name="value">value by which the bound counter metric should be added.</param>
-        public abstract void Add(in SpanContext context, T value);
+        public abstract void Add(in SpanReference context, T value);
```

Comment:
should the parameters be called reference or context?

Reply:
My vote would be `reference`

Reply:
or `spanReference`.
Either way, please do it consistently across this whole PR.

Reply:
in many places its "ctx". We cannot replace with "ref" as its reserved word.
Probably `spanReference` or `spanRef` for short?

Reply:
thanks! changing to `spanReference`